k8s/helm/values.yaml

```
@@ -0,0 +1,56 @@
+ingress-nginx:
+  controller:
+    ingressClass: k8s-ecosystem-ces-service
+    service:
+      external:
+        enabled: false
+      ports:
+        http: 80
+        https: 443
+    scope:
+      enabled: true
+      namespace: "ecosystem"
```

```suggestion
      namespace: "$(POD_NAMESPACE)"
```

I think this could solve the problem with the Jenkins-Pipeline which deploys to another namespace